### PR TITLE
[WIP] 3.4 blog post

### PR DESCRIPTION
**Pull request by MelSumner**
TODO:
- [x] Ember JS changes
- [x] Ember Data changes
- [x] Ember CLI changes
- [x] added note about 3.4 becoming LTS

### CODE REVIEW

**Comment by kennethlarsen** [comment]
I'm gonna use this comment to collect things that should be included in the release post (will update as I find things).

**Deprecations**
* Use closure actions instead of `sendAction` https://deprecations-app-prod.herokuapp.com/deprecations/v3.x/#toc_ember-component-send-action

**New Features**
* angle bracket support

**Comment by kategengler** [comment]
Reminder that 3.4 becomes LTS when 3.5 is released. Here's an old blog post from a similar scenario https://emberjs.com/blog/2017/03/19/ember-2-12-released.html#toc_changes-in-ember-js-2-12

Also 3.4 is the last version of Ember that will work with the polyfill addon for features that were deprecated in 2.x https://github.com/emberjs/ember-2-legacy

**Comment by Kerrick** [comment]
It could also be worth mentioning that 3.4 will embrace the autorun: https://github.com/rwjblue/rfcs/pull/1#issuecomment-419105847

**Comment by jrowlingson** [comment]
In the angle bracket section, it might be worth mentioning a reminder on the property lookup fallback behavior for folks [converting](https://github.com/emberjs/ember.js/pull/16991) over.

**Comment by dbbk** [comment]
What's happening with this? 3.4 was released a month ago.

File: source/blog/2018-08-27-ember-3-4-released.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+</SuperSelect>
+```
+
+**It's important to note that the classic invocation syntax is not deprecated in favour of this new invocation.** You're still free to use the classic invocation syntax; but users should be aware that angle bracket invocation does have a few advantages.
+
+The main advantage of the angle bracket invocation syntax is clarity. Because component invocation is often encapsulating important pieces of UI, a dedicated syntax would help visually distinguish them from other handlebars constructs, such as control flow and dynamic values. This can be seen in the example shown above – the angle bracket syntax made it very easy to see the component invocations as well as the `{{#each}}` loop, especially with syntax highlight.
+
+Guides will be updated to reflect the new syntax in the coming weeks.
+
+Custom component manager (2 of 2)
```
Comment (jessica-jordan) [inline source/blog/2018-08-27-ember-3-4-released.md]: It seems the custom component manager would need another quick write up

Comment (jessica-jordan) [reply source/blog/2018-08-27-ember-3-4-released.md]: https://github.com/emberjs/website/pull/3617

**Pull request merged**